android/gltfio-android/src/main/java/com/google/android/filament/gltfio/AssetLoader.java

```diff
@@ -54,11 +53,21 @@
  *         assetLoader.createAssetFromBinary(ByteBuffer.wrap(bytes))!!
  *     }
  *
- *     ResourceLoader(engine).loadResources(filamentAsset).destroy()
+ *     val resourceLoader = ResourceLoader(engine)
+ *     resourceLoader.loadResources(filamentAsset)
+ *     for (uri in filamentAsset.resourceUris) {
```

Comment:
Does this line really work?  Looks like `getResourceUris` returns void, not an array.

Reply:
Yeah, I pasted the snippet in my activity to verify. Only the native method returns `void` since you pass in the output array. The java methods returns a `String[]`

Reply:
Thanks for testing this!